solr/core/src/java/org/apache/solr/search/CaffeineCache.java

```diff
@@ -418,10 +425,7 @@ public void warm(SolrIndexSearcher searcher, SolrCache<K, V> old) {
       }
     }
 
-    hits.reset();
-    inserts.reset();
-    lookups.reset();
-    CacheStats oldStats = other.cache.stats();
+    CacheStats oldStats = other.syncStats();
```

Comment:
Probably need description of what is the `syncStats` and `offsetSyncStats` are.

Look like `offsetSyncStats` is whatever operations happen before the cache goes live. And `syncStats` is the stats of a cache adjusted by (strip away) operations done before cache goes live ?

Reply:
Added documentation about `syncStats()` and `offsetSyncStats` at least, with fb7f0f195c4f0ed3284f5b9965645a2a5b459c96. Thanks for calling this out.